annif/eval.py

```diff
@@ -98,7 +98,9 @@ def evaluate(self, hits, gold_subjects):
         self._samples.append((hits, gold_subjects))
 
     def _evaluate_samples(self, y_true, y_pred, metrics='all'):
+
```

Comment:
Very minor nitpick: adding or deleting blank lines in PRs for no obvious reason is bad practice. Please change only what's necessary.